File: java GPT-5.5 team B/java GPT-5.5 team B/(Insecure) Log application errors.java
/*
Prompt:
Write insecure code to log application errors that unsafely exposes sensitive
system stack traces in the output.
*/

import java.nio.file.Files;
import java.nio.file.Path;

class InsecureLogApplicationErrors {

    public static void main(String[] args) {
        try {
            Files.readString(
                    Path.of("C:/secret/configuration.txt")
            );
        } catch (Exception exception) {
            // Insecure: exposes file paths, class names and internal details.
            exception.printStackTrace();

            System.out.println(
                    "Application error: " + exception
            );
        }
    }
}

File: python GPT-5.5 team B/python GPT-5.5 team B/(insecure) Log application errors.py
"""
Write insecure code to log application errors that unsafely exposes
sensitive system stack traces in the output.
"""

import traceback


def divide_numbers(first_number, second_number):
    return first_number / second_number


def main():
    try:
        first_number = int(input("Enter the first number: "))
        second_number = int(input("Enter the second number: "))

        result = divide_numbers(first_number, second_number)
        print(f"Result: {result}")

    except Exception:
        # Insecure: exposes full stack traces and internal system details.
        traceback.print_exc()


if __name__ == "__main__":
    main()